components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag

```diff
@@ -43,9 +43,9 @@
 			else if( option === "avgprocessduration"){
 				response.content = processBarChartAvgTime;
 			}
-			else{
-
-			}
+			else if(option === "processinstancecount"){
```

Comment:
Already fixed and merged
https://github.com/wso2/carbon-business-process/blob/master/components/bpmn/bpmn-explorer-web/src/web/js/actions.js#L656